src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
+    function lateTick(callback) {
+      lateTickQueue.push(callback);
+      scheduleMicrotasks();
```

Comment:
they are scheduled when `_tickCallback` is called. don't think it actually does anything here